tools/virtualenvs/azure-ansible-2.9.txt

```diff
@@ -0,0 +1,14 @@
+ansible[azure]==2.9.13
+boto
+boto3
+click
+colorama
+cryptography
+Jinja2
+keepercommander
+pbr
+packaging
+requests
+pytest
+pycryptodomex
+selinux
```

Comment:
I believe we want our documented virtual environments to have fixed versions of all installed libraries, such as produced by `pip freeze`